packages/zoe/src/contracts/sellItems.js

```diff
@@ -29,11 +29,14 @@ import '../../exported';
  *
  * @type {ContractStartFn}
  */
-const start = (zcf, { pricePerItem }) => {
+const start = zcf => {
+  const { pricePerItem, issuers, maths } = zcf.getTerms();
   const allKeywords = ['Items', 'Money'];
   assertIssuerKeywords(zcf, harden(allKeywords));
   assertNatMathHelpers(zcf, pricePerItem.brand);
 
+  const { Money: moneyMath, Items: itemsMath } = maths;
```

Comment:
Why not destructure this in place on line 33? AFAICT you have no other need for the variable `maths`